source/_components/emulated_hue.markdown

```diff
@@ -106,7 +106,7 @@ emulated_hue:
 The following are attributes that can be applied in the `entities` section:
 
 - **name** (*Optional*): The name that the emulated Hue will use. The default for this is the entity's friendly name.
-- **hidden** (*Optional*): Whether or not the emulated Hue bridge should expose the entity. Adding `hidden: false` will expose the entity to Alexa. The default value for this attribute is controlled by the `expose_by_default` option.
+- **hidden** (*Optional*): Whether or not the emulated Hue bridge should expose the entity. Adding `hidden: false` will expose the entity to Alexa. If you set `expose_by_default` on false, you need to set `hidden: false` to expose the entity. The default value for this attribute is controlled by the `expose_by_default` option.
```

Comment:
So this is already in the current version:

>  Adding `hidden: false` will expose the entity to Alexa.

And now you are adding this to the next sentence?

> If you set `expose_by_default` on false, you need to set `hidden: false` to expose the entity.

I'm sorry, the first covers the addition right?